gpuarray/src/private_config.h

```diff
@@ -0,0 +1,53 @@
+/******************************************************************
```

Comment:
I forgot to update .gitgnore while moving it. I rebase interactively. What should be done with the comments in #else?